docs/synapse/userguides/userguide_section5.rst

```diff
@@ -0,0 +1,43 @@
+
+Data Model - Node Concepts
+==========================
+
+Recall from `Data Model – Basics`__ that the key data model components within Synapse are **nodes** and **tags.** Broadly speaking:
+
+- Nodes commonly represent "facts" or "observables": things that are objectively true or verifiable.
+
+- Tags commonly represent information that may be more changeable over time. Most commonly, tags represent "analytical assessments": conclusions drawn from observables that may change if data or the assessment of the data changes. In some cases, tags represent “facts” that may change over time, such as the geolocation of an IP address (represented by a location tag [``#loc``]) or the association of an IP address with a specialized service, such as a TOR exit node or a VPN provider.
```

Comment:
I think we should remove the `[]` that surrounds the tag name to match formatting of the rest of the document.